index.bs

```diff
@@ -165,6 +165,42 @@ the user-agent should retain the previously computed layout state if possible.
   [=skipped=] is a rendering performance optimization.
 </div>
 
+Accessibility {#accessibility}
+=============
+
+Similar to the way 'subtree-visibility' affects painted output of the element's
+subtree, it also affects information exposed to the
+<a href="https://w3c.github.io/css-aam/#dfn-accessibility-tree">accessibility
+tree</a> and <a href="https://w3c.github.io/css-aam/#dfn-assistive-technology">
+assistive technologies</a>:
+* The structure of the page should be exposed to assistive technologies.
+* When [=skipped=], the element's subtree should be exposed as
+    <a href="https://w3c.github.io/css-aam/#dfn-hidden">hidden</a>.
+
+<div class=note>
+  The general idea for exposing elements as hidden is the following: if content
```

Comment:
Suggested rephrasing:

Since assistive technologies may provide quick access to offscreen elements, it is important that elements in `subtree-visibility: auto` subtrees be made available to assistive technologies to provide this functionality, since they are intended to be observable to users.

Conversely, since elements in `subtree-visibility: hidden` subtrees are not intended to be observable by users, they should not be exposed to assistive technology.

Reply:
This particular phrasing seems to contradict the normative text. The problem is specifically about the "auto" value. When off-screen, the elements are not rendered (ie they are 'skipped'). So they should be excluded from the accessibility tree (according to the bullet points above).

Here, however, we are saying that should be included for quick off-screen access. I think what's missing is the distinction between exposing the element vs exposing the layout information (size and positioning). 

I think we should expose the element (which is what I meant by the 'structure of the page'). But we can't expose the layout information, since that would cause rendering to me noticeably slower. 

Do you know if there's terminology here that we can use to convey this?

Reply:
> When off-screen, the elements are not rendered (ie they are 'skipped'). So they should be excluded from the accessibility tree (according to the bullet points above).

Which bullet points?

> I think we should expose the element (which is what I meant by the 'structure of the page'). 

Hm, I found that wording hard to follow. It's ambiguous, I think.

> But we can't expose the layout information, since that would cause rendering to me noticeably slower.

Agreed.

> Do you know if there's terminology here that we can use to convey this?

I thought it was covered by the paragraph below.

Reply:
> > When off-screen, the elements are not rendered (ie they are 'skipped'). So they should be excluded from the accessibility tree (according to the bullet points above).
> 
> Which bullet points?

Sorry, I was referencing the bullet points you suggested in the previous comment. Specifically the second one says:

* Subtrees with a subtree-visibility value of skipped should be excluded from the accessibility tree.

"skipped" as we defined it in the spec means not-rendered: this would be, among others, the off-screen elements with `subtree-visibility: auto`. So, that bullet point says don't include off-screen elements with `subtree-visibility: auto` in the accessibility tree. 

However, the suggested phrasing of this paragraph says that off-screen elements with `subtree-visibility: auto` should be made available to the assistive technologies.

Doesn't that mean we should include them in the accessibility tree? Or is there a distinction between including something in the accessibility tree vs making it available to assistive technology?

Reply:
I've rephrased this a bit, new text: https://vmpstr.github.io/display-locking/#accessibility 

Let me know if that reads ok, or whether I should make more changes. I'd rather get this text correct and have it be unambiguous

Reply:
I think the new phrasing looks great! Let's commit this PR. Alice, if you have more comments please add them and we can add them in a new PR.